ext/liquid_c/context.c

```diff
@@ -142,6 +164,7 @@ void init_liquid_context()
     id_ivar_environments = rb_intern("@environments");
     id_ivar_static_environments = rb_intern("@static_environments");
     id_ivar_strict_variables = rb_intern("@strict_variables");
+    id_ivar_document_body = rb_intern("@document_body");
```

Comment:
Are we OK with ruby code being able to set or remove this instance variable?  If not, then we could keep it internal to C code by removing the `@` prefix to the instance variable name.